### ISSUE TITLE
Should app.js be in version control?

### ISSUE THREAD

**Issue by morty** [datagovuk/apiserver#36]
It seems to be a compiled resource which would suggest it should not be in version control.

**Comment by rossjones** [comment]
Yes and no.  Normally all the JS in web/static/js is compiled to priv/static/app.js (as ES6), and the rest in web/static/vendor (ES <6)  into vendor.js - we probably shouldn't have priv/ in git at all. This only happens in MIX_ENV=dev though, not when MIX_ENV=prod.  We're running this way as we're not doing proper releases yet, we're just pulling from git on prod.

When we implement exrm we can move to just building properly and it'll end up in the .tgz release so we won't need priv.
